app/cafe.py

```diff
@@ -0,0 +1,21 @@
+import datetime
+from app.errors import (NotVaccinatedError,
+                        OutdatedVaccineError,
+                        NotWearingMaskError)
+
+
+class Cafe:
+    def __init__(self, name: str):
+        self.name = name
+
+    def visit_cafe(self, visitor: dict):
+        if not("vaccine" in visitor):
+            raise \
+                NotVaccinatedError(f"{visitor['name']} is not vaccinated")
```

Comment:
better not transfer lines with `\`
better follow this style:
```
raise NotVaccinatedError(
    f"{visitor['name']} is not vaccinated"
)
```
Fix this throughout your code